ui/src/containers/SolutionList.js

```diff
@@ -42,56 +105,289 @@ const SolutionsList = props => {
             {version.version}
           </VersionLabel>
         )),
-      flexGrow: 1
+      flexGrow: 1,
     },
+  ];
+
+  const environmentsColumn = [
     {
-      label: intl.messages.deployed_version,
-      dataKey: 'versions',
-      renderer: versions =>
-        versions.map((version, index) => {
-          return version.deployed && version.ui_url ? (
-            <a href={version.ui_url} key={`deployed_version_${index}`}>
-              {version.version}
-            </a>
-          ) : null;
-        }),
-      flexGrow: 1
-    }
+      label: intl.messages.name,
+      dataKey: 'name',
+    },
+    {
+      label: intl.messages.description,
+      dataKey: 'description',
+      flexGrow: 1,
+    },
+    {
+      label: intl.messages.solutions,
+      dataKey: 'solutions',
+      renderer: (solutions, row) => {
+        return (
+          <div>
+            <span>
+              {solutions.map((solution, idx) => {
+                const solutionRow = sortedSolutions.find(
+                  s => s.name === solution.name,
+                );
+
+                const solutionVersion = solutionRow?.versions?.find(
+                  v => v.version === solution.version,
+                );
+
+                return (
+                  <ButtonContainer
+                    key={`solution_${idx}`}
+                    marginLeft={idx !== 0}
+                  >
+                    <Button
+                      size="smaller"
+                      text={`${solution.name} ${solution.version}`}
+                      icon={<i className="fas fa-external-link-alt" />}
+                      onClick={() => {
+                        if (solutionRow && solutionVersion) {
```

Comment:
I do not know why you use ```solutionRow```here.
We should check ```solutionVersion.ui_url``` to display this button or not. It means the solution UI is running. Otherwise, we can go to a wrong url